src/test/java/com/exacttarget/fuelsdk/ETClientTest.java

```diff
@@ -34,35 +34,65 @@
 
 package com.exacttarget.fuelsdk;
 
+import java.net.URL;
 import java.text.DateFormat;
 import java.text.SimpleDateFormat;
+import java.util.Properties;
 
-import org.junit.Assume;
-import org.junit.BeforeClass;
 import org.junit.Test;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 
 public class ETClientTest {
-    
-/*    public static void main(String[] args){
-        ETClientTest etc = new ETClientTest();
-        try{
-            etc.testBackwardCompatibility1();
-        }catch(ETSdkException ex){
-            ex.printStackTrace();
+
+    @Test
+    public void testNoDefaultForEndpoint() throws ETSdkException
+    {
+        URL fileToRead = ETClientTest.class.getClassLoader().getResource("noendpoint.properties");
```

Comment:
@ipetresf 
I see the below 3 lines in all the tests, why not move this to a separate function and just pass the property name ?
and call that function inside the try of each test.

URL fileToRead = ETClientTest.class.getClassLoader().getResource("noendpoint.properties");
ETConfiguration stubConfig = new ETConfiguration(fileToRead.getFile());
ETClient client = new ETClient(stubConfig);